### ISSUE TITLE
Section 2(a) unnecessarily uses term "intellectual property"

### ISSUE THREAD

**Issue by mdekstrand** [twitter/innovators-patent-agreement#1]
Section 2(a) in the IPA unnecessarily uses the term "intellectual property lawsuit" in a way that does not seem to add anything to the clearer statement "patent or copyright lawsuit".

Given that "intellectual property" doesn't provide any particular benefit in this statement, and that there are potential problems with the term, especially down the road as many people - of which it seems that Twitter is a part - push for reform in how copyrights and patents are handled in the US (see http://techliberation.com/2006/11/21/why-i-dont-say-intellectual-property-and-you-shouldnt-either/ for a non-Richard-Stallman argument against the term), it seems to me that that the agreement would be just as effective, clearer, and of more long-term benefit if section 2(a) listed the specific bodies of law under which defensive patent use is justified rather than the vague, catch-all term.

**Comment by benltwitter** [comment]
Fair point.  Pondering whether a term other than "patent infringement lawsuit" makes sense here.
